SUMMARY.md

```diff
@@ -147,8 +147,9 @@
     * [storeConfiguration](collector/remote-control/apis/storeconfiguration.md)
     * [getFromStore](collector/remote-control/apis/getfromstore.md)
   * [Sources](collector/remote-control/sources.md)
-  * [Requests](collector/remote-control/requests.md)
+  * [Object Explorer](collector/remote-control/object-explorer.md)
   * [WITSML Backlog Sync](collector/remote-control/backlog-sync.md)
+  * [Properties](collector/remote-control/properties.md)
```

Comment:
Missing `witsml-requests.md`, `opc-requests.md` at this point in the SUMMARY as explained below (in another comment).

Reply:
Fixed in the commit 658fb61c6ec0b99b0e364406743bb74a265617ab